static/templates/settings/organization_settings_admin.hbs

```diff
@@ -81,7 +81,7 @@
             <div class="inline-block organization-settings-parent">
                 <div class="input-group">
                     <label for="id_realm_message_retention_setting" class="dropdown-title">{{t "Message retention period" }}
-                        <i class="fa fa-info-circle settings-info-icon realm_message_retention_tooltip tippy-zulip-tooltip"
+                        <i class="fa fa-question-circle-o settings-question-icon realm_message_retention_tooltip tippy-zulip-tooltip"
```

Comment:
Why we are not using the filled icon here? As per the pattern I notice, we are using filled icons for tooltips and empty icons for links.

Reply:
Ohk. I thought I read that somewhere but was not sure and also there are still some question marks with filled-in background which confused me further.

Reply:
@alya what do you think of using filled icons for tooltips and empty icons for links? I personally think it makes sense to differentiate the two, but if you prefer otherwise I can replace all filled icons with empty ones